packages/components/text-field/src/text-field.ts

```diff
@@ -151,6 +151,20 @@ export class TextField extends FormControlMixin(ScopedElementsMixin(LitElement))
     `;
   }
 
+  override getLocalizedValidationMessage(): string {
+    if (this.validity.tooShort) {
+      const length = this.value.length;
+
+      return msg(
+        str`Please enter at least ${this.minLength} characters (you currently have ${length} character${
+          length > 1 ? 's' : ''
```

Comment:
Will this message also be show when the user hasn't entered any characters at all?
Because then it should be `length === 1 ? '' : 's'`
Because it is "0 characterS"

Reply:
No, `tooShort` validation only applies with there at least 1 character. This is the native browser behavior.